src/Renci.SshNet/Security/Cryptography/EcdsaDigitalSignature.cs

```diff
@@ -39,7 +39,7 @@ public override bool Verify(byte[] input, byte[] signature)
             // for 521 sig_size is 132
             var sig_size = _key.KeyLength == 521 ? 132 : _key.KeyLength / 4;
             var ssh_data = new SshDataSignature(signature, sig_size);
-#if NETSTANDARD2_0
+#if NETSTANDARD
```

Comment:
Could you change this to ```#if NETSTANDARD2_1_OR_GREATER || NET5_0_OR_GREATER```, so it won't be included for NETSTANDARD1_3 (OK, not really an issue, but as long as we are here...) but it will be included for net5.0 targets (futureproofing)?

Reply:
I propose to reverse the checks (here and elsewhere):

**Now**
```cs
#if NETSTANDARD2_0
            return _key.Ecdsa.VerifyData(input, ssh_data.Signature, _key.HashAlgorithm);
#else
            var ecdsa = (ECDsaCng)_key.Ecdsa;
            ecdsa.HashAlgorithm = _key.HashAlgorithm;
            return ecdsa.VerifyData(input, ssh_data.Signature);
#endif
```

**Proposal**
```cs
#if NETFRAMEWORK
            var ecdsa = (ECDsaCng)_key.Ecdsa;
            ecdsa.HashAlgorithm = _key.HashAlgorithm;
            return ecdsa.VerifyData(input, ssh_data.Signature);
#else
            return _key.Ecdsa.VerifyData(input, ssh_data.Signature, _key.HashAlgorithm);
#endif
```